contactSalon.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./reset.css">
+  <link rel="stylesheet" href="./contactSalon.css"
+  >
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Document</title>
+</head>
+<body>
+  <header>
+    <img src="./ZIZURZ LOGO.png" class="logo" alt="ZIZURZ LOGO">
+<nav>
+  <a class="has-border" href="./landingSalon.html">HOME</a>
+  <a class="has-border" href="./aboutSalon.html">ABOUT</a>
+  <a class="has-border" href="./gallerySalon.html">HAIRSTYLES</a>
+  <a class="has-border" href="./news.html">NEWS</a>
+  <a class="has-border" href="./contactSalon.html">CONTACT</a>
+
+</nav>
+</header>
+<div class="bulk">
+  <h1>Contact Us</h1><br><br>
+      <p>Want an easier solution for a Free Website? Head straight to Wix and immediately start customizing your website! Wix is an online website builder with a simple drag & drop interface, meaning you do the work online and instantly publish to the web. Nothing to download, nothing to upload. All Wix templates are fully customizable and free to use. Just pick one you like, click Edit, and enter the online editor. Change, add, and remove items as you like. Wix also offers a ton of free design elements right inside the editor, like images, icons, animation files, and interactive widgets. Publish your Free Website in minutes!</p>
+      <br><br>
+      <p>
+        Address : 7177 Saint Michael Drive, Los Angeles, CA 90048, United States <br>
+        Office Phone Number : (864) 232-3553 <br>
+        Fax Number : (864) 232-4160 <br>
+        Email : info@companyname.com <br>
+      </p>
```

Comment:
this works but you could also make each line a `p` tag. it'll automatically break `<br>` because `p` elements are block elements and won't allow anything next to it. 
`<p>Address : 7177 Saint Michael Drive, Los Angeles, CA 90048, United States</p>`
`<p>Office Phone Number : (864) 232-3553</p>`
`<p>Fax Number : (864) 232-4160</p>`
and then use margin top/bottom to add some space between them. no longer need to use `<br>`